commit 9aaeafa70c5126b22857485449f8d5eeb1d9c527
Author: Carl Love <cel@us.ibm.com>
Date:   Tue Sep 7 17:15:38 2021 +0000

    Update the expected output for test_isa_3_1_VRT.
    
    The inputs into the vinsdlx,vinsdrx instructions changed as a result of
    the impossible constraint issue fix.  This patch updates the expected
    results.
    
    https://bugs.kde.org/show_bug.cgi?id=441534

diff --git a/none/tests/ppc64/test_isa_3_1_VRT.stdout.exp b/none/tests/ppc64/test_isa_3_1_VRT.stdout.exp
index 39c4d8619..8f04ee2cb 100644
--- a/none/tests/ppc64/test_isa_3_1_VRT.stdout.exp
+++ b/none/tests/ppc64/test_isa_3_1_VRT.stdout.exp
@@ -6798,24 +6798,24 @@ vinsdlx 0 4 =>                4,1111111111111111
 vinsdlx 0 6 =>                6,1111111111111111
 vinsdlx 0 8 =>                8,1111111111111111
 vinsdlx 0 a =>                a,1111111111111111
-vinsdlx 4 0 => 1111111100000000,0000000011111111
-vinsdlx 4 2 => 1111111100000000,0000000211111111
-vinsdlx 4 4 => 1111111100000000,0000000411111111
-vinsdlx 4 6 => 1111111100000000,0000000611111111
-vinsdlx 4 8 => 1111111100000000,0000000811111111
-vinsdlx 4 a => 1111111100000000,0000000a11111111
-vinsdlx 8 0 => 1111111111111111,0000000000000000
-vinsdlx 8 2 => 1111111111111111,0000000000000002
-vinsdlx 8 4 => 1111111111111111,0000000000000004
-vinsdlx 8 6 => 1111111111111111,0000000000000006
-vinsdlx 8 8 => 1111111111111111,0000000000000008
-vinsdlx 8 a => 1111111111111111,000000000000000a
-vinsdlx 3 0 => 1111110000000000,0000001111111111
-vinsdlx 3 2 => 1111110000000000,0000021111111111
-vinsdlx 3 4 => 1111110000000000,0000041111111111
-vinsdlx 3 6 => 1111110000000000,0000061111111111
-vinsdlx 3 8 => 1111110000000000,0000081111111111
-vinsdlx 3 a => 1111110000000000,00000a1111111111
+vinsdlx 6 0 => 1111111111110000,0000000000001111
+vinsdlx 6 2 => 1111111111110000,0000000000021111
+vinsdlx 6 4 => 1111111111110000,0000000000041111
+vinsdlx 6 6 => 1111111111110000,0000000000061111
+vinsdlx 6 8 => 1111111111110000,0000000000081111
+vinsdlx 6 a => 1111111111110000,00000000000a1111
+vinsdlx 6 0 => 1111111111110000,0000000000001111
+vinsdlx 6 2 => 1111111111110000,0000000000021111
+vinsdlx 6 4 => 1111111111110000,0000000000041111
+vinsdlx 6 6 => 1111111111110000,0000000000061111
+vinsdlx 6 8 => 1111111111110000,0000000000081111
+vinsdlx 6 a => 1111111111110000,00000000000a1111
+vinsdlx 2 0 => 1111000000000000,0000111111111111
+vinsdlx 2 2 => 1111000000000000,0002111111111111
+vinsdlx 2 4 => 1111000000000000,0004111111111111
+vinsdlx 2 6 => 1111000000000000,0006111111111111
+vinsdlx 2 8 => 1111000000000000,0008111111111111
+vinsdlx 2 a => 1111000000000000,000a111111111111
 
 vinsdrx 0 0 => 1111111111111111,0000000000000000
 vinsdrx 0 2 => 1111111111111111,0000000000000002
@@ -6823,24 +6823,24 @@ vinsdrx 0 4 => 1111111111111111,0000000000000004
 vinsdrx 0 6 => 1111111111111111,0000000000000006
 vinsdrx 0 8 => 1111111111111111,0000000000000008
 vinsdrx 0 a => 1111111111111111,000000000000000a
-vinsdrx 4 0 => 1111111100000000,0000000011111111
-vinsdrx 4 2 => 1111111100000000,0000000211111111
-vinsdrx 4 4 => 1111111100000000,0000000411111111
-vinsdrx 4 6 => 1111111100000000,0000000611111111
-vinsdrx 4 8 => 1111111100000000,0000000811111111
-vinsdrx 4 a => 1111111100000000,0000000a11111111
-vinsdrx 8 0 =>                0,1111111111111111
-vinsdrx 8 2 =>                2,1111111111111111
-vinsdrx 8 4 =>                4,1111111111111111
-vinsdrx 8 6 =>                6,1111111111111111
-vinsdrx 8 8 =>                8,1111111111111111
-vinsdrx 8 a =>                a,1111111111111111
-vinsdrx 3 0 => 1111111111000000,0000000000111111
-vinsdrx 3 2 => 1111111111000000,0000000002111111
-vinsdrx 3 4 => 1111111111000000,0000000004111111
-vinsdrx 3 6 => 1111111111000000,0000000006111111
-vinsdrx 3 8 => 1111111111000000,0000000008111111
-vinsdrx 3 a => 1111111111000000,000000000a111111
+vinsdrx 6 0 => 1111000000000000,0000111111111111
+vinsdrx 6 2 => 1111000000000000,0002111111111111
+vinsdrx 6 4 => 1111000000000000,0004111111111111
+vinsdrx 6 6 => 1111000000000000,0006111111111111
+vinsdrx 6 8 => 1111000000000000,0008111111111111
+vinsdrx 6 a => 1111000000000000,000a111111111111
+vinsdrx 6 0 => 1111000000000000,0000111111111111
+vinsdrx 6 2 => 1111000000000000,0002111111111111
+vinsdrx 6 4 => 1111000000000000,0004111111111111
+vinsdrx 6 6 => 1111000000000000,0006111111111111
+vinsdrx 6 8 => 1111000000000000,0008111111111111
+vinsdrx 6 a => 1111000000000000,000a111111111111
+vinsdrx 2 0 => 1111111111110000,0000000000001111
+vinsdrx 2 2 => 1111111111110000,0000000000021111
+vinsdrx 2 4 => 1111111111110000,0000000000041111
+vinsdrx 2 6 => 1111111111110000,0000000000061111
+vinsdrx 2 8 => 1111111111110000,0000000000081111
+vinsdrx 2 a => 1111111111110000,00000000000a1111
 
 vinsd 3 0 => 1111110000000000,0000001111111111
 vinsd 3 2 => 1111110000000000,0000021111111111
